tutorials/terraform/mes-migration.tf

```
@@ -5,11 +5,11 @@
 
 # Specify the pre-installation parameters
 locals {
-  folder_id          = "" # Your Folder ID.
-  mes_admin_password = "" # Administrator password for Managed Service for Elasticsearch cluster.
-  mes_edition        = "" # Managed Service for Elasticsearch destination cluster edition, Basic or Platinum. See https://cloud.yandex.ru/docs/managed-elasticsearch/concepts/es-editions.
-  mes_version        = "" # Managed Service for Elasticsearch destination cluster version, should be equal or newer than source cluster version.
-  bucket_name        = "" # Object Storage bucket name. Should be unique in Cloud.
+  folder_id          = "" # Your folder ID.
+  mes_admin_password = "" # Administrator password for the Managed Service for Elasticsearch cluster.
+  mes_edition        = "" # The Managed Service for Elasticsearch destination cluster edition, Basic or Platinum. See https://cloud.yandex.ru/docs/managed-elasticsearch/concepts/es-editions.
+  mes_version        = "" # The Managed Service for Elasticsearch destination cluster version, should be equal or newer than the source cluster version.
```

Тут аналогично предыдущему, но дополнительно лучше разбить сложное предложение на два и для обозначения более поздней версии использовать later (так рекомендуют все стайлгайды):
```suggestion
  mes_version        = "" # Version of the Managed Service for Elasticsearch target cluster. Should be the same as the source cluster version or later.
```